src/Fieldtypes/ResponsiveFieldtype.php

```diff
@@ -173,6 +173,10 @@ public function preProcessIndex($data)
 
     public function process($data)
     {
+        if(! is_iterable($data) ) {
+            return array();
```

Comment:
Can you change this to `return [];` instead?

Reply:
done

Reply:
bumping this...